src/main/java/mesosphere/marathon/client/model/v2/App.java

```diff
@@ -9,25 +9,26 @@
 import mesosphere.marathon.client.utils.ModelUtils;
 
 public class App {
-	private String id;
-	private String cmd;
-	private String version;
-	private Integer instances;
-	private Double cpus;
-	private Double mem;
-	private Collection<String> uris;
-	private List<List<String>> constraints;
-	private Container container;
-	private Map<String, String> env;
-	private Map<String, String> labels;
-	private String executor;
-	private List<Integer> ports;
-	private Collection<Task> tasks;
-	private Integer tasksStaged;
-	private Integer tasksRunning;
+    private String id;
```

Comment:
It's not _my_ branch, but I understand what you mean. When editing these files—even with their maddening formatting—I've tried to adhere to the "When in Rome, do as the Romans do" maxim: try to match what's there, and change as little as possible.